documents/Libraries/sxpbrlib/sxpbrlib_defs.mtlx

```diff
@@ -53,7 +53,7 @@
     <input name="tangent" type="vector3">
       <geomprop name="tangent" type="vector3" space="world"/>
     </input>
-    <input name="distribution" type="string" value="ggx"/>
+    <input name="distribution" type="string" value="ggx" enum="ggx"/>
```

Comment:
Not sure what all the options are but at least it's mappable with an enum.

Reply:
Ah nice! Currently we only support GGX, but we might add Beckman and other distributions later.